TestDB.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import sys; sys.path.append('../')
+import faker
+
+class TestUser(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+        self.faker = faker.Faker()
+        faker = self.faker
+
+    def test_save_customer(self):
+        customer = Customer()
+        customer.first_name = faker.first_name()
+        customer.last_name = faker.last_name()
+        customer.date_created = faker.date()
+        user = User()
+        customer = user.save(customer)
```

Comment:
We were thinking of creating one save method flexible enough to 1.) receive data from a class (customer, for instance) and 2.) translate that data into an insert statement. The generic insert statement would adapt depending on the properties passed to it. Thus, when the save method is called for a customer, it would collect everything from name to date_created and pass that to the correct table in the database. Likewise, when the save method is called for an order, it would take the properties of an order and pass them to the order table on the database. Perhaps that's naive. One solution we're considering is creating a parent class qua model with a save function that other classes can inherit (customer, for instance). Thus, "customer" would inherit the save method from "model" and add its individual properties.